web/files/assets/css/custom.css

```diff
@@ -38,26 +38,121 @@
 
 /* Tables */
 table {
-  padding: 0; }
-  table tr {
+    padding: 0;
+}
+table tr {
     border-top: 1px solid #cccccc;
     background-color: white;
     margin: 0;
-    padding: 0; }
-    table tr:nth-child(2n) {
-      background-color: #f8f8f8; }
-    table tr th {
-      font-weight: bold;
-      border: 1px solid #cccccc;
-      text-align: left;
-      margin: 0;
-      padding: 6px 13px; }
-    table tr td {
-      border: 1px solid #cccccc;
-      text-align: left;
-      margin: 0;
-      padding: 6px 13px; }
-    table tr th :first-child, table tr td :first-child {
-      margin-top: 0; }
-    table tr th :last-child, table tr td :last-child {
-      margin-bottom: 0; }
+    padding: 0;
+}
+table tr:nth-child(2n) {
+    background-color: #f8f8f8;
+}
+table tr th {
+    font-weight: bold;
+    border: 1px solid #cccccc;
+    text-align: left;
+    margin: 0;
+    padding: 6px 13px;
+}
+table tr td {
+    border: 1px solid #cccccc;
+    text-align: left;
+    margin: 0;
+    padding: 6px 13px;
+}
+table tr th :first-child, table tr td :first-child {
+    margin-top: 0;
+}
+table tr th :last-child, table tr td :last-child {
+    margin-bottom: 0;
+}
+
+/* These containers were inspired/modified from the sphinx-gallery
+ * gallery.css.
+ */
+/*.plasmapy-feature-container {*/
+/*    background: #fff;*/
+/*    border: solid #fff 1px;*/
+/*    -moz-border-radius: 5px;*/
+/*    -webkit-border-radius: 5px;*/
+/*    border-radius: 5px;*/
+/*    box-shadow: none;*/
+/*    float: left;*/
+/*    margin: 5px;*/
+/*    min-height: 230px;*/
+/*    padding-top: 5px;*/
+/*    position: relative;*/
+/*}*/
+/*.plasmapy-feature-container:hover {*/
+/*  border: solid #b4ddfc 1px;*/
+/*  box-shadow: 0 0 15px rgba(142, 176, 202, 0.5);*/
+/*}*/
+
+* {
+    box-sizing: border-box;
+}
+
+/* Float three columns side by side */
+.feature-column {
+    float: left;
+    width: 33%;
+    padding: 0 5px;
+}
+
+/* Remove extra left and right margins, due to padding */
+.feature-row {margin: 0 -5px}
+
+/* Clear floats after the columns */
+.feature-row:after {
+    content: "";
+    display: table;
+    clear: both;
+}
+
+/* Responsive columns...3 columns move to 3 rows when columns are less
+ * than 600px
+ */
+@media screen and (max-width: 800px) {
+    .feature-column {
+        width: 100%;
+    }
+}
+
+.feature-card {
+    position: relative;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+    padding: 16px;
+    background-color: #f1f1f1;
+    border-style: solid;
+    border-color: #f1f1f1;
+    border-width: 2px;
+    height: 200px;
+
+    background-image: linear-gradient(rgba(255, 255, 255, 0.5), rgba(255, 255, 255, 0.5)), url("/images/favicon-128.thumbnail.png");
```

Comment:
Ah, I was wondering why it looked a bit low res - maybe use https://github.com/PlasmaPy/PlasmaPy-logo/blob/master/exports/graphic-circular.png instead?

Reply:
yes, that is the reason